doc-content/enterprise-only/installation/clustering-download-eap-bc-proc.adoc

```diff
@@ -17,6 +17,6 @@ Download and install {EAP} {EAP_VERSION} and {PRODUCT} {PRODUCT_VERSION} on each
 * *Version: {PRODUCT_VERSION}*
 .. Download  *{PRODUCT} {PRODUCT_VERSION_LONG}* {CENTRAL} Deployable for {EAP} 7 (`{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-{URL_COMPONENT_CENTRAL}-eap7-deployable.zip`).
 . Extract the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-{URL_COMPONENT_CENTRAL}-eap7-deployable.zip` file to a temporary directory. In the following commands this directory is called `__TEMP_DIR__`.
-. Copy the contents of `_TEMP_DIR_/{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-{URL_COMPONENT_CENTRAL}-eap7-deployable/jboss-eap-7.3` to `_EAP_HOME_`.
+. Copy the contents of `_TEMP_DIR_/{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-{URL_COMPONENT_CENTRAL}-eap7-deployable/jboss-eap-{EAP_VERSION}` to `_EAP_HOME_`.
 . Download and apply the latest {PRODUCT} patch, if available.
 . Configure Red Hat Single Sign-On for your high availability environment. For more information, see {URL_INTEGRATING}#assembly-integrating-sso[_{INTEGRATING_SSO}_] and the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/{RH-SSO_VERSION}/html-single/server_administration_guide/[_Red Hat Single Sign-On Server Administration Guide_].
```

Comment:
Use variable :RH-SSO: Red Hat Single Sign-On. And search for. I know this isn't text that you changed but would be good to do it now. Up to you.